GRIB2_CodeFlag_4_2_CodeTable_en.csv

```diff
@@ -666,6 +666,16 @@ Parameter number by product discipline and parameter category,"Product disciplin
 Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",23-191,,Reserved,,,,Operational
 Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",192-254,,Reserved for local use,,,,Operational
 Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",255,,Missing,,,,Operational
+Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 22: drought indices",0,,Standard Precipitation Index (SPI),,,dimensionless,Operational
+Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 22: drought indices",1,,Standardised Precipitation Evapotranspiration Index (SPEI),,,dimensionless,Operational
+Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 22: drought indices",2,,Standardised Streamflow index (SFI),,,dimensionless,Operational
```

Comment:
The acronym in the WMO Guide is "SSFI"

Reply:
for standardised streamflow index? We can change it to match the WMO guide. I will mention it to the researchers developing this to check with them

Reply:
yes.

Reply:
@sebvi I assume that it's ok to change this from SFI to SSFI